resources/js/modding-profile/main.coffee

```diff
@@ -44,6 +46,7 @@ export class Main extends React.PureComponent
     @cache = {}
     @tabs = React.createRef()
     @pages = React.createRef()
+    @reviewsConfig = Object.assign({}, props.reviewsConfig, disableInsert: true)
```

Comment:
that's an attempt but pasting still works (and this also removes insert paragraph button)